packages/ui/src/Common.ts

```diff
@@ -1908,6 +1910,18 @@ export interface TableProps {
    * Extra controls to render below the table next to pagination
    */
   extraControls?: React.ReactElement;
+  /**
+   * Pinned rows to the left of the table
```

Comment:
Not exactly sure what this means. I see that the header is pinned but where are rows getting pinned?

Reply:
Ahh they're not. I was trying to support arbitrary rows and columns being pinned, but it was even more complex than this, so I left it at "the header is pinned, you can pin columns." Removed!